enterprise/frontend/src/embedding-sdk/components/private/Breadcrumb/SdkBreadcrumbs.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from "react";
+import { P, match } from "ts-pattern";
+
+import { InteractiveDashboard } from "embedding-sdk/components/public";
+import { useSdkBreadcrumbs } from "embedding-sdk/hooks/private/use-sdk-breadcrumb";
+import { CommonSdkStoryWrapper } from "embedding-sdk/test/CommonSdkStoryWrapper";
+import type { SdkCollectionId } from "embedding-sdk/types";
+import type { SdkBreadcrumbItemType } from "embedding-sdk/types/breadcrumb";
+import { Stack } from "metabase/ui";
+
+import { CollectionBrowser } from "../../public/CollectionBrowser";
+import { InteractiveQuestion } from "../../public/InteractiveQuestion";
+
+import { SdkBreadcrumbs } from "./SdkBreadcrumbs";
+import { SdkBreadcrumbsProvider } from "./SdkBreadcrumbsProvider";
+
+export default {
+  title: "EmbeddingSDK/SdkBreadcrumb",
+  component: SdkBreadcrumbs,
+  parameters: {
+    layout: "fullscreen",
+  },
+  decorators: [CommonSdkStoryWrapper],
+};
+
+type View =
+  | { type: "collection"; id: SdkCollectionId }
+  | { type: Exclude<SdkBreadcrumbItemType, "collection">; id: string | number };
+
+const SdkBreadcrumbStory = () => {
+  const { currentLocation } = useSdkBreadcrumbs();
+
+  const [view, setView] = useState<View>({ type: "collection", id: "root" });
+
+  // If a user clicks on a collection breadcrumb, switch the view.
+  useEffect(() => {
+    if (currentLocation?.type === "collection") {
+      setView({ type: currentLocation.type, id: currentLocation.id });
+    }
+  }, [currentLocation]);
+
+  const viewContent = match(view)
+    .with({ type: "collection" }, () => (
+      <CollectionBrowser
+        collectionId={view.id}
+        onClick={(item) => {
+          const type = match<string, SdkBreadcrumbItemType>(item.model)
+            .with("card", () => "question")
+            .with("dataset", () => "model")
+            .otherwise((model) => model as SdkBreadcrumbItemType);
+
+          setView({ type, id: item.id });
+        }}
+      />
+    ))
+    .with({ type: "dashboard" }, () => (
+      <InteractiveDashboard dashboardId={view.id} />
+    ))
+    .with({ type: P.union("question", "metric", "model") }, () => (
+      <InteractiveQuestion questionId={view.id} />
+    ))
+    .exhaustive();
+
+  return (
+    <Stack p="md" gap="sm">
```

Comment:
Seems the `gap` prop does  not have any effect visually:

<img width="412" height="742" alt="Image" src="https://github.com/user-attachments/assets/6e270e8e-8792-473a-93db-67fa52737a08" />

Reply:
Fixed in https://github.com/metabase/metabase/pull/62128/commits/e87a489c6ffe8ed802cb37ed18dfbf0bc56e5753, thanks!

<img width="756" height="1074" alt="CleanShot 2568-08-16 at 03 13 31@2x" src="https://github.com/user-attachments/assets/038e947e-cccd-4ab9-a1bc-3863ff503b75" />